components/lib/dataprotect/src/main/java/mozilla/components/lib/dataprotect/Keystore.kt

```diff
@@ -212,6 +212,7 @@ open class Keystore(
      * @throws GeneralSecurityException If the data could not be encrypted
      */
     @Throws(GeneralSecurityException::class)
+    @Synchronized
```

Comment:
here, could just `synchronize` over the three `val` declarations - don't need it for the final `return` since it doesn't access the underlying `cipher` instance